src/uv-common.c

```diff
@@ -67,6 +67,14 @@ static uv__allocator_t uv__allocator = {
   free,
 };
 
+char* uv__strdup(const char* s) {
```

Comment:
@yichoi Yes. the binary size matter. But The ``uv__strdup`` code came from uv original source. I would not choose moving the code to unix/darwin.c. I will keep it in original location, but enclose the definition with ``__APPLE__ `` flags.